### PR TITLE
Add HTMLy

### PR DESCRIPTION
**Pull request by bttrx**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/htmly.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+source_code_url: "https://github.com/danpros/htmly/"
+description: "HTMLy is an open source databaseless PHP blogging platform. A flat-file CMS that allows you to create a fast, secure, and powerful website or blog in seconds."
+licenses:
+  - GPL-2.0
+platforms:
+  - PHP
+tags:
+  - Blogging Platforms
+  - Content Management Systems (CMS)
+depends_3rdparty: false
```
Comment (kokomo123) [inline software/htmly.yml:12]:
```suggestion
```
This is a redundant line, its already set to false 👍🏻

Comment (bttrx) [reply software/htmly.yml:12]: You are right. I didn't understand the explanation in the template.

File: software/htmly.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: "HTMLy"
+website_url: "https://www.htmly.com/"
+source_code_url: "https://github.com/danpros/htmly/"
+description: "HTMLy is an open source databaseless PHP blogging platform. A flat-file CMS that allows you to create a fast, secure, and powerful website or blog in seconds."
```
Comment (nodiscc) [inline software/htmly.yml:4]:
```suggestion
description: "Databaseless PHP blogging platform. A flat-file CMS that allows you to create a fast, secure, and powerful website or blog in seconds."
```

Improved description, as per https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines

File: software/htmly.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: "HTMLy"
+website_url: "https://www.htmly.com/"
+source_code_url: "https://github.com/danpros/htmly/"
```
Comment (nodiscc) [inline software/htmly.yml:3]:
```suggestion
source_code_url: "https://github.com/danpros/htmly"
```

Not sure if this is important, however small fix for consistency with other entries

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by kokomo123 — APPROVED** [review]
LGTM, no issues here. Thanks for the fixes!

**Review by nodiscc — APPROVED** [review]

**Pull request merged**